app/components/new-stepper.hbs

```diff
@@ -13,6 +23,38 @@
           @disabled={{not this.joinApplicationTerms.hasUserAcceptedTerms}}
         />
       </div>
+
+      {{else if (eq this.currentStep 1)}}
+        <NewJoinSteps::NewStepOne
+          @setIsPreValid={{this.setIsPreValid}}
+          @isValid={{this.isValid}}
+          @setIsValid={{this.setIsValid}}
+        />
     {{/if}}
   </form>
+
+  {{#if (not-eq this.currentStep this.MIN_STEP)}}
+    <div class="new-stepper__buttons">
+      {{#if this.showPreviousButton}}
+        <Reusables::Button
+          @text="Previous"
+          @variant="light"
+          @test="prev-btn"
+          @type="button"
+          @onClick={{this.decrementStep}}
+        />
+      {{else}}
+        <div></div>
+      {{/if}}
+
+      <Reusables::Button
+        @text="Next"
+        @variant="dark"
+        @test="next-btn"
+        @type="button"
+        @onClick={{this.incrementStep}}
+        @disabled={{not (or this.preValid this.isValid)}}
```

Comment:
This reads as "disabled if NOT (preValid OR isValid)" which means "disabled if both are false". Consider extracting to a getter for clarity

get isNextButtonDisabled() {
  return !(this.preValid || this.isValid);
}